openshift/cronJob-template.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  annotations:
+    thoth-station.ninja/template-version: 0.7.0
+  labels:
+    app: srcopsmetrics
+  name: srcopsmetrics
+spec:
+  schedule: "0 0 * * 0"
+  jobTemplate:
+    metadata:
+      name: srcopsmetrics
+    spec:
+      template:
+        spec:
+          containers:
+            - name: srcopsmetrics
+              image: docker-registry.default.svc:5000/dtuchyna-thoth-dev/srcopsmetrics
```

Comment:
I don't think you have to provide all this, `srcopsmetrics` image will be pulled by default by the `docker-registry` in `dtuchyna-thoth-dev` if you deployed in that namespace.

Reply:
when line 19 commented out, openshift console has problem
```The CronJob "srcopsmetrics" is invalid: spec.jobTemplate.spec.template.spec.containers[0].image: Required value```

Reply:
Right, let's add a parameter in the template:

Reply:
added to all templates